client/payment-methods/index.js

```diff
@@ -88,51 +94,72 @@ const PaymentMethods = ( { enabledMethodIds, onEnabledMethodIdsChange } ) => {
 		}
 	};
 
+	const handleAddPaymentMethod = () => {
+		setShowPaymentMethodsSelector( true );
+	};
+
+	const renderPaymentMethodSelector = () => {
```

Comment:
(minor) it seems that using a function to be called when `showPaymentMethodsSelector` is truthy makes the body of `PaymentMethods` a bit longer - I'm more used to having most of the JSX at the `return` 😄  this is ok, tho!
There seem to be a slight inconsistency - the `handleAddPaymentMethod` is extracted to a separate function, but the `onClose` handler passed to `PaymentMethodsSelector` isn't.

But again, I'm being pretty ⛏️  😄

Reply:
:smile: no worries at all, you're right your suggestions make the code more consistent and I appreciate that! I'll update the code with your suggestions :+1: